main.ts

```diff
@@ -70,10 +70,12 @@ if (!running) {
 
 async function onReady() {
   if (cmd.length) {
-    const process = Deno.run({ cmd })
-    const status = await process.status()
+    const [commandPath, ...args] = cmd
+    const command = new Deno.Command(commandPath!, { args })
+    const process = command.spawn()
+    process.unref()
```

Comment:
I don't think this is right... we explicitly want to keep this process open until the inner one closes

Reply:
removed